pkg/zng/docs/spec.md

```diff
@@ -95,9 +95,9 @@ logs from the open-source Zeek system might look like this
 ```
 #alias:addr=ip
 #24:record[_path:string,ts:time,uid:string,id:record[orig_h:addr,orig_p:port,resp_h:addr,resp_p:port]...
-#25:record[_path:string,ts:time,uid:string,id:record[orig_h:addr,orig_p:port,resp_h:addr,resp_p:port]...
-24:conn;1425565514.419939;CogZFI3py5JsFZGik;[192.168.1.1:;80/tcp;192.168.1.2;8080;]...
-25:dns;1425565514.419987;CogZFI3py5JsFZGik;[192.168.1.1:;5353/udp;192.168.1.2;5353;]...
+#25:record[_path:string,ts:time,fuid:string,tx_hosts:set[addr]...
+24:[conn;1425565514.419939;CogZFI3py5JsFZGik;[192.168.1.1:;80/tcp;192.168.1.2;8080;]...
+25:[files;1425565514.419987;Fj8sRF1gdneMHN700d;[52.218.49.89;52.218.48.169;]...
```

Comment:
The original example that showed `conn` and `dns` was showing the same initial few fields. I figured using `files` was an improvement since it shows real-world differences in those first few columns of the Zeek log, a different composite type in the form of `set`, etc. Also, getting the `files` output from `zq` led me to notice the absence of the `[` in the `conn` record value.